### PR TITLE
CRM-18516 Convert the date fields relating to group caching & acl cac…

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
…hing timestamp

### CODE REVIEW

File: CRM/ACL/BAO/Cache.php [hunk 1]
```diff
@@ -149,14 +149,11 @@ public static function resetCache() {
...
 
-    //make sure to give original timezone settings again.
-    $now = CRM_Utils_Date::getUTCTime();
-
     $query = "
 DELETE
 FROM   civicrm_acl_cache
 WHERE  modified_date IS NULL
-   OR  (TIMESTAMPDIFF(MINUTE, modified_date, $now) >= $smartGroupCacheTimeout)
+   OR  (TIMESTAMPDIFF(MINUTE, modified_date, NOW())) >= $smartGroupCacheTimeout)
```
Comment (seamuslee001) [inline CRM/ACL/BAO/Cache.php]: @eileenmcnaughton i think you got 1 too many ) in this.

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/ACL/BAO/Cache.php [hunk 2]
```diff
@@ -149,14 +149,11 @@ public static function resetCache() {
...
 
-    //make sure to give original timezone settings again.
-    $now = CRM_Utils_Date::getUTCTime();
-
     $query = "
 DELETE
 FROM   civicrm_acl_cache
 WHERE  modified_date IS NULL
-   OR  (TIMESTAMPDIFF(MINUTE, modified_date, $now) >= $smartGroupCacheTimeout)
+   OR  (TIMESTAMPDIFF(MINUTE, modified_date, NOW()) >= $smartGroupCacheTimeout)
```
Comment (JKingsnorth) [inline CRM/ACL/BAO/Cache.php]: Doesn't keeping this make it an unindexed query? Something we were trying to get rid of here: https://github.com/civicrm/civicrm-core/pull/7906/files#diff-e71b99945ad53aa0cfbd9efd2dbfc168L78

We can calculate that in PHP instead? Or is that out of scope for this PR?

Comment (eileenmcnaughton) [inline CRM/ACL/BAO/Cache.php]: yeah I went further & added a whole lot more in this one https://github.com/civicrm/civicrm-core/pull/8370

**Comment by eileenmcnaughton** [comment]
I'm closing this in favour of #8370  which contains a more complete solution

**Pull request closed without merging**